app/utilities/schema.py

```diff
@@ -177,7 +178,44 @@ def load_schema_from_url(schema_url, language_code):
 
     constructed_schema_url = f"{schema_url}?language={language_code}"
 
-    req = requests.get(constructed_schema_url)
+    session = requests.Session()
+
+    retries = Retry(
+        total=3,
+        allowed_methods=frozenset({"GET"}),
+        backoff_factor=0.1,
+        status_forcelist=[
+            408,
+            429,
+            500,
+            501,
+            502,
+            503,
+            504,
+            505,
+            506,
+            507,
+            509,
+            510,
+            511,
```

Comment:
Not all these are retriable. We should only need 500, 502, 503, 504.

Reply:
OK, so I only left the ones that are retriable, thanks.